main/guides/getting-started/swapaoo-making-a-payment-explainer.md

```diff
@@ -0,0 +1,41 @@
+# Making Payments in Agoric Smart Contracts
+If you're new to developing smart contracts on the Agoric platform, one of the key concepts you'll encounter is making payments between parties. In this document, we'll explain how to make a payment using deposit facets in an Agoric smart contract.
+
+## Code Snippet
+Let's take a look at the following code snippet from the Swaparoo contract:
+```js
+const secondDepositFacet = await E(depositFacetFromAddr).lookup(
+  secondPartyAddress,
+  'depositFacet',
+);
+
+await E(secondDepositFacet).receive(secondSeatInvitation);
+
+return 'invitation sent';
+```
+
+## Step-by-Step Explanation
+### Retrieving the Deposit Facet:
+- `depositFacetFromAddr` is an object that provides a lookup function for deposit facets associated with addresses.
+- The lookup function is called with `secondPartyAddress` and `'depositFacet'` as arguments to retrieve the deposit facet associated with the `secondPartyAddress`.
+- Since `lookup` is an asynchronous function that returns a promise, the `await` keyword is used to wait for the promise to resolve.
+- The resulting deposit facet is stored in the `secondDepositFacet` variable.
+
+### Making the Payment:
+- `secondDepositFacet` represents the deposit facet obtained in the previous step.
+- The `receive` method is called on `secondDepositFacet`, passing `secondSeatInvitation` as an argument.
+- `secondSeatInvitation` is likely a payment or a representation of an offer or seat in the Zoe contract framework.
+- Since `receive` is another asynchronous operation, the `await` keyword is again used to wait for it to complete.
+- By calling `receive` on the deposit facet with `secondSeatInvitation`, the payment or offer represented by `secondSeatInvitation` is effectively transferred or deposited into the account associated with `secondDepositFacet`.
+
+### Returning a Result:
+- After the payment has been successfully made by calling `receive`, the function returns the string `'invitation sent'` to indicate that the invitation has been sent.
+
+## Deposit Facets in Agoric
+In the Agoric smart contract framework, deposit facets are used as a secure way to transfer and manage digital assets and payments between parties. By calling the receive method on a deposit facet and passing in a payment or offer, the smart contract can safely deposit or transfer assets into the account associated with that facet.
+
+Deposit facets provide an abstraction layer for handling payments and ensure that the transfers are performed securely and reliably within the smart contract.
+
+## Video Walkthrough
+As you're going through this tutorial it may be helpful to watch this video walkthrough.
+<iframe width="560" height="315" src="https://www.youtube.com/embed/XeHBMO7SckU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
The video seems OK; it doesn't have the above problems of emphasis.

It's missing "how did I get namesByAddress in the first place?" though.

Reply:
Does the written explainer cover this enough, or should I reshoot the video?